app/models/ta.rb

```diff
@@ -14,6 +14,14 @@ class Ta < User
 
   has_many :grade_entry_student_tas
   has_many :grade_entry_students, through: :grade_entry_student_tas
+  
+  def get_num_assigned(assignment)
+    assignment.ta_memberships.where(user_id: id).size
+  end
+
+  def get_num_marked(assignment)
+    (assignment.ta_memberships.where(user_id: id).select {|x| x.grouping.marking_completed?}).size
```

Comment:
Line is too long. [98/80]<br>Space between { and | missing.<br>Space missing inside }.